implementing-nosto/implement-search/getting-started-with-nosto-search/implementing-category-pages.md

```diff
@@ -33,9 +32,9 @@ init({
 ```
 {% endcode %}
 
-#### categoryQuery parameter as function&#x20;
+#### Category query parameter as function&#x20;
 
-In the example above, we supply autocomplete query parameters as an object. Additionally, categoryQuery parameters can also be supplied as a function. The function flavor can be used for building complex query parameters and provides access to other pre-defined configuration parameters. Section below shows an example of categoryQuery supplied as a function which provides the  product variationId by accessing the pre-defined categoryId() and categoryPath() methods from the default configuration.
+In the example above, we supply autocomplete query parameters as an object. Additionally, `categoryQuery` parameter can also be supplied as a function. The function flavor can be used for building complex query parameters and provides access to other pre-defined configuration parameters. Section below shows an example of `categoryQuery` supplied as a function which provides the  product `variationId` by accessing the pre-defined `categoryId` and `categoryPath` methods from the default configuration.
```

Comment:
Is there an extra space between "provides the" and the "product"?

Reply:
fixed